### ISSUE TITLE
Clearer Error message on comment

### ISSUE THREAD

**Issue by GrahamSH-LLK** [LLK/scratch-www#4116]
If you get logged out automatically during writing a comment, it gives a generic error message.  (Oops! Something went wrong posting your comment) It even looks like you are still logged in. You have to reload to find out that you are logged out, and your comment is lost. (Unless you copied it)

### Steps to Reproduce
Go to a project
Duplicate the tab
Start Writing a comment
Log Out in the other one
Go back to the other
See error message
This also happens when you are logged out automatically to save server usage.
### Operating System and Browser
ChromeOS Chrome

**Comment by Explosion-Scratch** [comment]
So are you proposing thbat it says something more specific? Like "Oop! You have been logged out! To prevent lost work your comment was copied to the clipboard."

**Comment by GrahamSH-LLK** [comment]
Yeah! I think something like "Oops! You have been logged out! To prevent lost work your comment was copied to the clipboard. Please reload to sign back in!" It would look like this: 
![image](https://user-images.githubusercontent.com/64214252/85025158-01d7c880-b145-11ea-877e-4e8c4aedfdfb.png)
We could also add a reload button.

**Comment by GrahamSH-LLK** [comment]
Ie. Something like this: 
![image](https://user-images.githubusercontent.com/64214252/85025813-df927a80-b145-11ea-842e-d333db8f2247.png) by changing `<span>Oops! Something went wrong posting your comment</span>` to `<span>Oops! You have been logged out! To prevent lost work your comment was copied to the clipboard. Please reload to sign back in! <a href="javascript:location.reload()"> Reload</a></span>
`

**Comment by BoomerScratch** [comment]
@GrahamSH-LLK I guess
```html
<span>Oops! You have been logged out! Please copy the comment to your clipboard to not lose your work. <a href="javascript:location.reload()">Reload</a> to refresh your session.</span>
```
is better?

**Comment by apple502j** [comment]
It's actually possible to automatically copy it to clipboard - but let's wait for the devs :P

**Comment by BoomerScratch** [comment]
@apple502j I know, but people might have other clipboard data, which will get overwritten. I guess adding a button is the best.

**Comment by GrahamSH-LLK** [comment]
Perhaps `<span>Oops! You have been logged out! To prevent lost work your comment was copied to the clipboard. Please <a href="javascript:location.reload()"> reload</a> and sign back in! </span>`
![image](https://user-images.githubusercontent.com/64214252/85029787-aa3c5b80-b14a-11ea-8c06-ffb8af192c5b.png)

**Comment by BoomerScratch** [comment]
@GrahamSH-LLK https://github.com/LLK/scratch-www/issues/4116#issuecomment-646027571
I think it's best not to automatically copy the comment to the clipboard - a button instead would be much better.

**Comment by GrahamSH-LLK** [comment]
Ok! Maybe something like this: `<span>Oops! You have been logged out! To prevent lost you can <a href="javascript:(function()%7Bvar%20copyText%20%3D%20document.getElementById(%22frc-compose-comment-3392903%22)%3BcopyText.select()%3BcopyText.setSelectionRange(0%2C%2099999)document.execCommand(%22copy%22)%7D)()"> copy</a>  your comment to the clipboard. Please <a href="javascript:location.reload()"> reload</a> and sign back in! 
</span>` 
![image](https://user-images.githubusercontent.com/64214252/85035621-639e2f80-b151-11ea-962c-3910ab3d59b2.png)
The code doesn't work yet though.

**Comment by BoomerScratch** [comment]
@GrahamSH-LLK Yes - that's better.

**Comment by benjiwheeler** [comment]
This would be good! And, it's currently a bit tricky to tell that you've been logged out -- we may need to do some work to make that easier to detect.

**Comment by Explosion-Scratch** [comment]
This sounds good but would'nt this change it fo every error? As benjiwheeler said, we would need to implement some sort of detection system in this case

**Comment by BoomerScratch** [comment]
@benjiwheeler Yes, it requires API changes. Else we can send a request to the session (https://scratch.mit.edu/session); the response will be 
```
{

}

```
when logged out.

**Comment by GrahamSH-LLK** [comment]
@BoomerScratch  @benjiwheeler  Would that work?

**Comment by benjiwheeler** [comment]
Something in that direction would work -- but since this connects to https://github.com/LLK/scratch-www/issues/2970, which is a big and complex issue, I think we should wait until we address that to address this one. I made a note on #2970 to mention this issue.
